### ISSUE TITLE
Caffe dropout_ratio is the same as TensorFlow keep_prob

### ISSUE THREAD

**Issue by chubei** [ethereon/caffe-tensorflow#110]
It seems Caffe `DropoutLayer` parameter `dropout_ratio` is directly used in TensorFlow `tf.nn.dropout` function as parameter `keep_prob`. Is this wrong?
